controller/backing_image_data_source_controller.go

```diff
@@ -836,14 +836,19 @@ func (c *BackingImageDataSourceController) prepareRunningParameters(bids *longho
 		}
 	}
 	if newSnapshotRequired {
+		freezeFilesystem, err := c.ds.GetFreezeFilesystemForSnapshotSetting(e)
+		if err != nil {
+			return err
+		}
```

Comment:
~If for some reason we can't get a true or false, would be better to default to "sync, don't freeze" rather than failing the operation?~
Never mind, I have my answer.  The datastore method only returns error on serious conditions like "can't find that volume."  Simple mechanical lookup errors will default to "false".